src/content/migrate/5.md

```diff
@@ -91,25 +78,27 @@ module.exports = {
 };
 ```
 
+You have to remove these options again when upgrading your configuration for webpack 5.
+
 T> webpack 5 removes these options from the configuration schema and will always use `false`.
 
-You have to remove these options again when upgrading your configuration for webpack 5.
+## Upgrade webpack to 5
 
-#### Upgrade webpack version
+Now let's upgrade webpack to version 5:
 
-npm: `npm install webpack@latest --dev`
+- npm: `npm install webpack@latest --dev`
 
-Yarn: `yarn add webpack@latest -D`
+- Yarn: `yarn add webpack@latest -D`
 
-#### Clean up configuration
+### Clean up configuration
 
 - Consider removing `optimization.moduleIds` and `optimization.chunkIds` from your webpack configuration. The defaults could be better, because they support long term caching in [`production mode`](/configuration/mode/#mode-production) and debugging in [`development` mode](/configuration/mode/#mode-development).
 - When using `[hash]` placeholder in webpack configuration, consider changing it to `[contenthash]`. It is not the same, but proven to be more effective.
 - If you are using Yarn's PnP and the `pnp-webpack-plugin`, we have good news: it is supported by default now. You have to remove it from the configuration.
 - If you are using `IgnorePlugin` with a regular expression as argument, it takes an `options` object now: `new IgnorePlugin({ resourceRegExp: /regExp/ })`.
-- If you are using `node.something: 'empty'` replace it with `resolve.fallback.something: false`.
+- If you are using something like `node.fs: 'empty'` replace it with `resolve.fallback.fs: false`.
```

Comment:
What about

```suggestion
- If you are using something like `node.{fs,path}: 'empty'` replace it with `resolve.fallback.{fs,path}: false`.
```

?

Reply:
I bet user would search `node.fs`.

Reply:
Why? It is that common?

Reply:
What I mean here is users would rather search `node.fs` than `node.{fs`. I don't expect google or algolia to find `node.{fs` for us when users type `node.fs`.

Reply:
oh yeah